### ISSUE TITLE
New update suggestion by jimonlinux for https://itsfoss.com/fix-sound-ubuntu-1304-quick-tip/

### ISSUE THREAD

**Issue by abhishekpc** [itsfoss/feedback#97]
am using Linux Leap 15.5.
I built a desktop computer on an Asus tuf b760-plus wifi gamming board.
I copied files from another computer which was working fine but, after everything was copied I could not get the sound to work on the new computer. After checking many times from the bios settings to sound software and even buying an asus sound card, I still couldn't get the sound to work.

One of the options in this web page suggested checking and replacing some hidden file, which jogged my memory. The new desktop permissions were set so that "root" was User/Owner and Group was "Trusted". This Desktop belonged to the Group "Trusted" and had the same permissions as root. The fix was to change permissions so that Owner/User "root" was replaced with the Login Name of the Desktop owner and Group remained unchanged.
To sum it up, add this to your list of things to check:
"CHECK PERMISSIONS".
jimonlinux
